cypress/support/index.js

```diff
@@ -17,7 +17,7 @@ import 'cypress-file-upload';
 import './data-import';
 import './commands';
 
-setInteractorTimeout(80_000);
+setInteractorTimeout(10_000);
```

Comment:
Why you change it? As I know we a tests that need this 80000 value

Reply:
yep, restore the previous version